bot/cogs/watchchannels/bigbrother.py

```diff
@@ -90,10 +100,8 @@ def __init__(self, bot: Bot) -> None:
 
         await ctx.send(msg)
 
-    @bigbrother_group.command(name='unwatch', aliases=('uw',))
-    @with_role(*MODERATION_ROLES)
-    async def unwatch_command(self, ctx: Context, user: FetchedMember, *, reason: str) -> None:
-        """Stop relaying messages by the given `user`."""
+    async def apply_unwatch(self, ctx: Context, user: FetchedMember, reason: str, banned: bool = False) -> None:
```

Comment:
I think a more generic name like `send_message` would be better. Regardless, this parameter should be documented.

Reply:
Addressed in most recent commit